CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 1.2.0-rc1 (2016-03-22)
+
+#### Scheduler
+
+- Move rescheduling out of exprimental
+- Differentiate constraint errors from affinity errors
+- Printing unsatisfiable constraints for container scheduling failure
+
+#### API
+
+- Introduce engine-api client to Swarm
+- Implement 'info' and 'version' with engine-api
+- Plumbing context through all calls to engine-api client
```

Comment:
I'll remove it. I think it's also included in "Introduce engine-api client to Swarm".